samples/subsys/usb/hid-mouse/README.rst

```diff
@@ -0,0 +1,70 @@
+.. _usb_hid-mouse:
+
+USB HID mouse Sample Application
+####################################
+
+Overview
+********
+
+This sample app demonstrates use of a USB HID driver provided by the zephyr
+project, enumerate a board into a mouse. But that mouse only has left button
+and can't move.
+This sample can be found under :file:`samples/subsys/usb/hid-mouse` in the
+Zephyr project tree.
+
+Requirements
+************
+
+This project requires an USB device driver, and there must has at least one
```

Comment:
If there's more than one button, how do you configure which button is the one used to simulate a left mouse button click?